index.html

```diff
@@ -29,5 +29,137 @@ <h1>Hey there. I'm Nilton<span> I'm a Full Stack web developer</span></h1>
           <li><a href="https://angel.co/u/nilton-segura"><img src="img/angellist.png"></a></li>
         </ul>         
     </div>
+    <a href="#"><img class="button" src="img/NormalButtonDisabled.png"></a>
+    <h2>My Recent Works</h2>
+    <img src="img/Indicator.png" alt="Litte divisor">
+    <div class="Works">
+      <div class="project">
```

Comment:
- [ ] Dear @Nkaleth, As the previous reviewer mentioned please make sure this section (the `about`) is a `section` tag instead of a `div`. This has a great benefit when it comes to semantics.
```html
<section class="Works">
```

Reply:
@Tadesse-Alemayehu, Thanks for the feedback.